homeassistant/components/gree/strings.json

```diff
@@ -4,10 +4,6 @@
       "confirm": {
         "description": "[%key:common::config_flow::description::confirm_setup%]"
       }
-    },
-    "abort": {
```

Comment:
Don't remove this. It's needed by the config flow.

Reply:
Hmmm... that wasn't intentional :S